scout/server/blueprints/variant/controllers.py

```diff
@@ -456,7 +456,7 @@ def get_loqusdb_obs_cases(store, variant_obj, category, obs_families=[]):
     return obs_cases
 
 
-def observations(store, loqusdb, case_obj, variant_obj):
+def observations(store, loqusdb, variant_obj):
     """Query observations for a variant.
 
     Check if variant_obj have been observed before ni the loqusdb instance.
```

Comment:
```suggestion
    Check if variant_obj have been observed before in the loqusdb instance.
```

Reply:
Also some of the following comments are for code that probably moved from here to `get_loqusdb_obs_cases()` - lets move them.

Reply:
Did you see this @northwestwitch? Those comments should fit well in the docstring of `get_loqusdb_obs_cases()`.